specification/search/data-plane/Azure.Search/preview/2020-06-30-Preview/searchindex.json

```diff
@@ -1057,6 +1210,14 @@
           "x-nullable": false,
           "description": "The relevance score of the document compared to other documents returned by the query."
         },
+        "@search.rerankerScore": {
+          "type": "number",
+          "format": "double",
```

Comment:
In regard to comment above, this is a double. Why isn't the other score a double?

Reply:
they should all be double